datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -5902,3 +5902,14 @@ ORDER BY k;
 ----
 1 1.8125 6.8007813 Float16 Float16
 2 8.5 8.5 Float16 Float16
+
+# issue: https://github.com/apache/datafusion/issues/12814 
```

Comment:
nit: this could be just above of
```
statement error DataFusion error: Error during planning: Aggregate functions are not allowed in the WHERE clause. Consider using HAVING instead
SELECT v1 FROM t1 WHERE ((count(v1) % 1) << 1) > 0;
```

so i doesn't "fload away" with new modifications to the file